sklearn/manifold/tests/test_locally_linear.py

```diff
@@ -33,7 +33,7 @@ def test_barycenter_kneighbors_graph():
     assert_less(linalg.norm(pred - X) / X.shape[0], 1)
 
 
-#----------------------------------------------------------------------
+# ----------------------------------------------------------------------
```

Comment:
I don't think this is really a relevant change; a better convention would be just `###########` or similar.